### PR TITLE
bpo-39481: PEP 585 for a variety of modules

### PR DESCRIPTION
**Pull request by isidentical**
- `queue.SimpleQueue`
- `queue.Queue`
- `tempfile.TemporaryDirectory`
- `unittest.case._AssertRaisesContext`

https://bugs.python.org/issue39481

### CODE REVIEW

File: Lib/test/test_genericalias.py [hunk 1]
```diff
@@ -35,7 +38,10 @@ def test_subscriptable(self):
                   Mapping, MutableMapping, MappingView,
                   KeysView, ItemsView, ValuesView,
                   Sequence, MutableSequence,
-                  MappingProxyType, DirEntry
+                  MappingProxyType, DirEntry,
+                  TemporaryDirectory, SpooledTemporaryFile
```
Comment (emmatyping) [inline Lib/test/test_genericalias.py]: You missed a comma here.

File: Lib/test/test_genericalias.py [hunk 2]
```diff
@@ -35,7 +38,10 @@ def test_subscriptable(self):
                   Mapping, MutableMapping, MappingView,
                   KeysView, ItemsView, ValuesView,
                   Sequence, MutableSequence,
-                  MappingProxyType, DirEntry
+                  MappingProxyType, DirEntry,
+                  TemporaryDirectory, SpooledTemporaryFile,
+                  Queue, SimpleQueue,
+                  _AssertRaisesContext
```
Comment (gvanrossum) [inline Lib/test/test_genericalias.py]: Trailing comma.
```suggestion
                  _AssertRaisesContext,
```

**Review by gvanrossum — APPROVED** [review]
Once your other PR (#19418) lands you will have to rebase; then ping me and I'll land it.

**Comment by isidentical** [comment]
For preventing future conflicts, appending group2 and 3s into this PR.

**Review by gvanrossum — APPROVED** [review]

**Pull request merged**